### ISSUE TITLE
BROKEN with last version of google-gax 

### ISSUE THREAD

**Issue by tcroiset** [GoogleCloudPlatform/google-cloud-python#2999]
with google-gax 0.15.5
(at least) pubsub publish doesn't work anymore:

```
16:54:55.000 File "/usr/local/lib/python2.7/dist-packages/google/cloud/pubsub/topic.py", line 249, in publish
16:54:55.000 api = client.publisher_api
16:54:55.000 File "/usr/local/lib/python2.7/dist-packages/google/cloud/pubsub/client.py", line 88, in publisher_api
16:54:55.000 generated = make_gax_publisher_api(self._connection)
16:54:55.000 File "/usr/local/lib/python2.7/dist-packages/google/cloud/pubsub/_gax.py", line 535, in make_gax_publisher_api
16:54:55.000 return PublisherClient(channel=channel)
16:54:55.000 File "/usr/local/lib/python2.7/dist-packages/google/cloud/gapic/pubsub/v1/publisher_client.py", line 188, in __init__
16:54:55.000 page_descriptors=self._PAGE_DESCRIPTORS)
16:54:55.000 File "/usr/local/lib/python2.7/dist-packages/google/gax/api_callable.py", line 361, in construct_settings
16:54:55.000 bundler = _construct_bundling(bundling_config, bundle_descriptor)
16:54:55.000 File "/usr/local/lib/python2.7/dist-packages/google/gax/api_callable.py", line 115, in _construct_bundling
16:54:55.000 bundler = gax.bundling.Executor(gax.BundleOptions(
16:54:55.000 AttributeError: 'module' object has no attribute 'bundling'
```

**Comment by dhermes** [comment]
@geigerj @lukesneeringer Any ideas?

**Comment by cuberskulk** [comment]
I can confirm. I have the same issue.

**Comment by miverson** [comment]
I've the same issue as well.

**Comment by dhermes** [comment]
Thanks guys. If you pin to an older version of `google-gax` does that fix the issue?

**Comment by miverson** [comment]
Yes.

**Comment by lukesneeringer** [comment]
I will have a fix out as soon as CI completes.

**Comment by lukesneeringer** [comment]
Fixed, and cut 0.15.6

**Comment by miverson** [comment]
Awesome! 

Thanks for jumping on this right away. You definitely improved my Monday.
